packages/core/admin/server/src/controllers/admin.ts

```diff
@@ -177,4 +177,170 @@ export default {
+        scheduledAt: {
+          $gt: new Date(),
+        },
+      },
+      sort: 'scheduledAt:asc',
+    });
+
+    const lastActivities = await strapi.documents('admin::audit-log').findMany({
+      limit: 10,
+      sort: 'createdAt:desc',
+    });
+
+    const oneWeekAgo = new Date();
+
+    oneWeekAgo.setDate(oneWeekAgo.getDate() - 7);
+
+    const contributions = await strapi.documents('admin::audit-log').findMany({
+      sort: 'date:desc',
+      populate: ['user'],
+      filters: {
+        action: ['entry.create', 'entry.update', 'entry.delete'],
+        date: {
+          $gt: oneWeekAgo,
+        },
+      },
+    });
+
+    const allContributors = contributions.reduce((acc, log) => {
+      const user = log.user.documentId;
+
+        if (!acc[user]) {
+          acc[user] = {
+            creations: 0,
+            updates: 0,
+            deletions: 0,
+            user: {
+              id: user,
+              firstname: log.user.firstname,
+              lastname: log.user.lastname,
+            },
+          };
+        }
+
+        if (log.action === 'entry.create') {
+          acc[user].creations += 1;
+        } else if (log.action === 'entry.update') {
+          acc[user].updates += 1;
+        } else if (log.action === 'entry.delete') {
+          acc[user].deletions += 1;
+        }
+
+        return acc;
+      },
+      {} as {
+        [id: string]: {
+          creations: number;
+          updates: number;
+          deletions: number;
+          user: { id: string; firstname: string; lastname: string };
+        };
+      }
+    );
+
+    const topContributors = Object.values(allContributors).sort((a, b) => {
+      const aTotal = a.creations + a.updates + a.deletions;
+      const bTotal = b.creations + b.updates + b.deletions;
+
+      return bTotal - aTotal;
+    }).slice(0, 5);
+
+
+    const locales = (await strapi.documents('plugin::i18n.locale').findMany()).reduce((acc, locale) => {
+      acc[locale.code] = locale.name;
+      return acc;
+    }, {} as Record<string, string>);
+
+    const assignedToMe = await assignedEntries(ctx.state.user.id, locales);
+
+
+    ctx.body = {
+      name: ctx.state.user.firstname,
+      statistics: stats,
+      releases: {
+        upcoming: upcomingReleases,
+      },
+      lastActivities,
+      topContributors: Object.values(topContributors),
+      assignedToMe,
+    };
+  },
 };
+
+type AssignedEntriesResult = {
+  contentType: { name: string, uid: string },
+  entry: { id: number, documentId: string, updatedAt: string, name: string, locale: string},
+}[];
+
+async function assignedEntries(userId: number, locales: Record<string, string>): Promise<AssignedEntriesResult> {
+  const workflows = await strapi.documents('plugin::review-workflows.workflow').findMany();
+
+  const contentTypes = [...new Set(workflows.flatMap((workflow) => workflow.contentTypes))]; // Set to remove duplicates
+
+  const result: AssignedEntriesResult = [];
+
+  await Promise.all(
```

Comment:
This would perform poorly in case the app has a lot content-types / entries. I think for a first iteration it would be better to not include this part.